torch_em/data/datasets/light_microscopy/hpa.py

```diff
@@ -370,7 +370,7 @@ def get_hpa_segmentation_dataset(
     Returns:
        The segmentation dataset.
     """
-    get_hpa_segmentation_data(path, download, channels, n_workers_preproc)
+    get_hpa_segmentation_data(path, download, n_workers_preproc)
```

Comment:
We should add a check here to ensure that `channels` passed by the user are valid.